components/team_sidebar/components/team_button.tsx

```diff
@@ -51,18 +50,12 @@ class TeamButton extends React.PureComponent<Props> {
         this.props.switchTeam(this.props.url);
     }
 
-    handleDisabled = (e: React.MouseEvent<HTMLAnchorElement, MouseEvent>) => {
-        e.preventDefault();
-    }
-
     render() {
         const {teamIconUrl, displayName, btnClass, mentions, unread, isDraggable = false, teamIndex, teamId, order} = this.props;
         const {formatMessage} = this.props.intl;
 
         let teamClass: string = this.props.active ? 'active' : '';
-        const disabled: string = this.props.disabled ? 'team-disabled' : '';
         const isNotCreateTeamButton: boolean = !this.props.url.endsWith('create_team') && !this.props.url.endsWith('select_team');
-        const handleClick = (this.props.active || this.props.disabled) ? this.handleDisabled : this.handleSwitch;
```

Comment:
Are these not required anymore?

Reply:
the disabled prop isn't used anywhere anymore xD